slf4j-propagation/src/main/java/nl/talsmasoftware/context/mdc/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2016-2021 Talsma ICT
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @deprecated This package was renamed to {@code nl.talsmasoftware.context.slf4j.mdc}
+ */
+package nl.talsmasoftware.context.mdc;
```

Comment:
Might be good to also mark the package as deprecated:
```suggestion
@Deprecated
package nl.talsmasoftware.context.mdc;
```

Reply:
@sjoerdtalsma, did you miss this comment?

Reply:
I did see the comment, but it didn't compile for me and I did not have time to find out why not, so I left it like this.
If you feel strongly about this, I welcome a (compiling) PR 😉

Reply:
Hmm yeah; handling of the `@Deprecated` annotation for packages is pretty inconsistent. It is probably fine like this since you also have annotated `MdcManager`.